rfcs/0087-aarch64-tier1.md

```diff
@@ -0,0 +1,96 @@
+---
+feature: aarch64-tier1
+start-date: 2021-03-09
+author: Vika Shleina
+co-authors: Graham Christensen
+shepherd-team: TBD
+shepherd-leader: TBD
+related-issues: TBD
+---
+
+# Summary
+[summary]: #summary
+
+Move `aarch64-linux` from a Tier 2 platform to Tier 1, as described in [RFC 0046](/rfcs/0046-platform-support-tiers.md)
+
+# Motivation
+[motivation]: #motivation
+
+`aarch64-linux` support in Nixpkgs and NixOS matures over time and becomes
+more and more stable, and more devices appear having NixOS on ARM support.
+Moving it to a Tier 1 platform will allow us to block release channels on
+aarch64-related build failures, making it easier and safer for ARM users
+to upgrade their systems, and will help in keeping software versions in
+sync between several architectures due to `x86_64-linux` and `aarch64-linux`
+builds sharing a channel.
+
+`aarch64-linux` will benefit from increased percieved binary cache coverage
+as an additional result of channel bumps waiting for aarch64 builds to finish,
+saving on build times for end users.
+
+## Prior art
+There were prior attempts at the same feat, but they failed due to technical
+limitations of Hydra:
+ - NixOS/nixpkgs@74c4e30 - disabled in 2017 because of memory issues
+ - NixOS/nixpkgs#52534, NixOS/nixpkgs@36a0c13 - re-enabled in 2018 to pre-build important outputs
+ - NixOS/nixpkgs@1bfe8f1 - disabled again due to hydra-evaluator issues
```

Comment:
I would say *partially reverted*, as we went from nothing to having limited support.

Relatedly, as a following paragraph in the prior art section, something along the line of.

Since https://github.com/NixOS/nixpkgs/commit/1bfe8f189bd25de73340635d4e94673000f273ec, in late 2018 NixOS already is blocking channel releases on the base system closure required to produce the installer image.